Libs/MRML/Core/vtkMRMLModelDisplayNode.cxx

```diff
@@ -31,20 +35,23 @@ vtkMRMLModelDisplayNode::vtkMRMLModelDisplayNode()
 {
   this->PassThrough = vtkPassThrough::New();
   this->AssignAttribute = vtkAssignAttribute::New();
+  this->ThresholdFilter = vtkThreshold::New();
+  this->Threshold = false;
```

Comment:
A name that includes "enabled" (ThresholdEnabled or EnableThreshold) would be more clear.